template/audit-lister.html

```diff
@@ -0,0 +1,78 @@
+<div id="{$_id}" class="{_ui}ui{/} {$class} {_class}feed{/}" style="{$style}" {$attributes}>
+{rows}
+    {row}
+    <div class="event">
+        <div class="label">
+            <i class="{icon}comment{/} icon"></i>
+        </div>
+        <div class="content">
+            <div class="summary">
+                <div class="user">{$user}</div>
+                {$action}
+                <div class="date">{$ts}</div>
+            </div>
+            <div class="meta">
+                {$descr}
+            </div>
+            <div class="extra text">
+                {changes}
+                    {change}
+
+                    <!-- option 1 - show as segment with field name in header and labels inside -->
```

Comment:
remove comments from {change}. Template is dumb and will add LOTS of those comments into output. 

Placing them between {changes} and {change} is best.

Reply:
fixed